package.json

```diff
@@ -21,10 +21,14 @@
     "type": "git",
     "url": "https://github.com/turbo87/sidebar-v2.git"
   },
+  "main": "./examples/ol5.js",
   "scripts": {
-    "test": "gulp lint"
+    "test": "gulp lint",
+    "start": "parcel ./examples/ol5.html",
+    "build": "parcel build --public-url ./ ./js/ol5-sidebar.js"
   },
   "devDependencies": {
+    "cssnano": "^4.1.3",
```

Comment:
what is this? why is this needed?

Reply:
Sincerely I don't know how it got inside! Removing it makes no difference. So, out!